.gitlab-ci.yml

```diff
@@ -1,10 +1,38 @@
-image: python:3.7.2-slim
+include:
+  - project: ultimaker/stardust/print-cluster-jedi-scripts
+    file: /gitlab/gitlab-ci-template.yml
+
+# Override these settings from the template because we don't have a k8s cluster available
+image: docker:19.03.1
+services:
+  - docker:19.03.1-dind
+
+.common:
+  tags:
+    - docker
+
+.build:
+  variables:
+    image: registry.gitlab.com/ultimaker/stardust/print-cluster-jedi-scripts/cmake:1.0.1
+    CMAKE_OPTIONS: "-DCMAKE_INSTALL_PREFIX=/usr -DEMBEDDED=ON"
+
+variables:
+  DOCKER_HOST: tcp://docker:2375
+  GIT_AUTHOR_NAME: embedded
+  GIT_AUTHOR_EMAIL: embedded@ultimaker.com
+  GIT_COMMITTER_NAME: embedded
+  GIT_COMMITTER_EMAIL: embedded@ultimaker.com
+  EMAIL: embedded@ultimaker.com
 
 stages:
   - validate
+  - test
+  - build
+  - release
 
 validate:
   stage: validate
+  image: python:3.7.2-slim
   script:
     - python -m pip install lxml==4.3.0
```

Comment:
Installing lxml takes quite some time, is there a way to cache it via a base image?